wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -14,15 +14,19 @@
     - `action_icon` - icon for the 'action' button, default is 'icon-plus'
 
 {% endcomment %}
-<header class="header {% if merged %}merged{% endif %} {% if search_form %}hasform{% endif %}">
+<header class="header{% if merged %} merged{% endif %}{% if search_form %}hasform{% endif %} {{ classname }}">
     {% block breadcrumb %}{% endblock %}
     <div class="row">
         <div class="left">
             <div class="col">
                 <h1 class="header__title">
-                    {% if icon %}{% icon name=icon %}{% endif %}
+                    {% if icon %}
+                        {% icon class_name="w-header__icon" name=icon %}
+                    {% elif avatar %}
+                        <div class="avatar"><img src="{% avatar_url user %}" alt="" /></div>{% endif %}
                     {{ title }}{% if subtitle %} <span class="header__subtitle">{{ subtitle }}</span>{% endif %}
```

Comment:
I recommend seeing how the username goes passed in to the `subtitle` instead. while this becomes part of the `h1` I think that kind of makes sense "Welcome to the blah blah LB Johnston".